src/model_service.hpp

```diff
@@ -52,4 +52,10 @@ class GetModelStatusImpl {
     static Status serializeResponse2Json(const tensorflow::serving::GetModelStatusResponse* response, std::string* output);
 };
 
+class GetModelsImpl {
```

Comment:
Do we actually need this class? Maybe those methods could go under `GetModelStatusImpl`?